### ISSUE TITLE
Generated examples don't include all attributes when an allOf also contains an anyOf/oneOf

### ISSUE THREAD

**Issue by gwerbin** [swagger-api/swagger-ui#7040]
### Q&A (please complete the following information)
 - OS: macOS
 - Browser: Chrome
 - Version: 88
 - Method of installation: N/A
 - Swagger-UI version: Using <https://editor.swagger.io>
 - Swagger/OpenAPI version: OpenAPI 3.0.3

### Content & configuration

Example Swagger/OpenAPI definition:

```yaml
openapi: 3.0.3
info:
  title: 'Things n stuff'
  version: 1.0.0

paths:
  '/data':
    get:
      responses:
        '200':
          description: 'The things you asked for.'
          content:
            application/json:
              schema:
                allOf:
                  - $ref: '#/components/schemas/PageInfo'
                  - type: object
                    required:
                      - items
                    properties:
                      items:
                        type: array
                        items:
                          allOf:
                            - anyOf:
                              - $ref: '#/components/schemas/Thing1'
                              - $ref: '#/components/schemas/Thing2'
                            - type: object
                              required:
                                - isImportant
                              properties:
                                isImportant:
                                  type: boolean

components:
  schemas:
    BaseThing:
      type: object
      required:
        - color
      properties:
        color:
          type: string

    Thing1:
      allOf:
        - $ref: '#/components/schemas/BaseThing'
        - type: object
          properties:
            color:
              type: string
              enum:
                - blue
                - red

    Thing2:
      allOf:
        - $ref: '#/components/schemas/BaseThing'
        - type: object
          properties:
            color:
              type: string
              enum:
                - green
                - yellow

    PageInfo:
      type: object
      properties:
        pageNumber:
          type: number
          minimum: 0
        totalPages:
          type: number
          minimum: 0
```

### Describe the bug you're encountering

The `isImportant` property is not added to the generated example. See the "screenshots" section.

The schema itself appears to be correct in the generated UI.

## To reproduce...

Copy and paste the schema above into <https://editor.swagger.io/>.

### Expected behavior

I expected the generated example in the screenshot to include `isImportant`.

### Screenshots

![](https://files.catbox.moe/ilq8x0.png)

### Additional context or thoughts

Thank you for the useful tool!

**Comment by mathis-m** [comment]
@gwerbin Thank you for reporting this. Good issue could find the problem instantly. Fix provided with #7041.

**Comment by gwerbin** [comment]
I actually forgot to upload the screenshot of the problematic generated example, but glad you were able to find it easily nonetheless!
